<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerMonitor</class>
 <widget class="QWidget" name="PowerMonitor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PowerMonitor</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>6</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 0);</string>
       </property>
       <property name="title">
        <string>显示面板</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="1">
               <layout class="QFormLayout" name="formLayout">
                <property name="verticalSpacing">
                 <number>25</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>波特率</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="comboBox_2">
                  <item>
                   <property name="text">
                    <string>1200</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2400</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4800</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19200</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>38400</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>57600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>115200</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>校验位</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="comboBox_3">
                  <item>
                   <property name="text">
                    <string>NONE</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>ODD</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>EVE</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>数据位</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QComboBox" name="comboBox_4">
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>停止位</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QComboBox" name="comboBox_5">
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0" rowspan="2">
               <widget class="QTextEdit" name="textEdit"/>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>打开串口</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="2">
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>电流</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_12">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_16">
                   <property name="horizontalSpacing">
                    <number>14</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>47</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>A 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit_7"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_14">
                     <property name="text">
                      <string>B 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_8"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_15">
                     <property name="text">
                      <string>C 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_9"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QGroupBox" name="groupBox_5">
                <property name="title">
                 <string>有功功率</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_19">
                   <property name="verticalSpacing">
                    <number>25</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_16">
                     <property name="text">
                      <string>A 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit_10"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_17">
                     <property name="text">
                      <string>B 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_11"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_18">
                     <property name="text">
                      <string>C 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_12"/>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_19">
                     <property name="text">
                      <string>总功率</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QLineEdit" name="lineEdit_13"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>电压</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_11">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_18">
                   <property name="horizontalSpacing">
                    <number>14</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>47</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_7">
                     <property name="text">
                      <string>A 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>B 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_2"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_9">
                     <property name="text">
                      <string>C 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_3"/>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="1">
                  <layout class="QGridLayout" name="gridLayout_17">
                   <property name="horizontalSpacing">
                    <number>14</number>
                   </property>
                   <property name="verticalSpacing">
                    <number>47</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>AB 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit_4"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>BC 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_5"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_12">
                     <property name="text">
                      <string>CA 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_6"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QGroupBox" name="groupBox_7">
                <property name="title">
                 <string>功率因数</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_15">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_21">
                   <property name="verticalSpacing">
                    <number>25</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_28">
                     <property name="text">
                      <string>A 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit_22"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_29">
                     <property name="text">
                      <string>B 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_23"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_30">
                     <property name="text">
                      <string>C 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_24"/>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_31">
                     <property name="text">
                      <string>总因数</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QLineEdit" name="lineEdit_25"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QGroupBox" name="groupBox_6">
                <property name="title">
                 <string>无功功率</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_14">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_20">
                   <property name="verticalSpacing">
                    <number>25</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_24">
                     <property name="text">
                      <string>A 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="lineEdit_18"/>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_25">
                     <property name="text">
                      <string>B 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="1">
                    <widget class="QLineEdit" name="lineEdit_19"/>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_26">
                     <property name="text">
                      <string>C 相</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="1">
                    <widget class="QLineEdit" name="lineEdit_20"/>
                   </item>
                   <item row="3" column="0">
                    <widget class="QLabel" name="label_27">
                     <property name="text">
                      <string>总功率</string>
                     </property>
                    </widget>
                   </item>
                   <item row="3" column="1">
                    <widget class="QLineEdit" name="lineEdit_21"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_3">
           <layout class="QGridLayout" name="gridLayout_22">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0" rowspan="6">
               <widget class="QwtPlot" name="plot">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="canvasBackground" stdset="0">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </brush>
                </property>
                <property name="autoReplot" stdset="0">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1" rowspan="6">
               <widget class="QSlider" name="slider_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QCheckBox" name="checkBox">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 85, 255);</string>
                </property>
                <property name="text">
                 <string>A相电压</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QCheckBox" name="checkBox_2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 0, 0);</string>
                </property>
                <property name="text">
                 <string>B相电压</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QCheckBox" name="checkBox_3">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 0);</string>
                </property>
                <property name="text">
                 <string>A相电流</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QCheckBox" name="checkBox_4">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 170, 0);</string>
                </property>
                <property name="text">
                 <string>B相电流</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QCheckBox" name="checkBox_5">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(170, 0, 255);</string>
                </property>
                <property name="text">
                 <string>总有功功率</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QCheckBox" name="checkBox_6">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(85, 170, 255);</string>
                </property>
                <property name="text">
                 <string>总功率因数</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QSlider" name="slider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4">
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tableWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>白天</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 255);</string>
       </property>
       <property name="text">
        <string>配电监控系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>选项</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPushButton" name="pushButton_4">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>连接</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>参数</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>曲线</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>报表</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
